src/becca/entities/bnote.ts

```diff
@@ -86,7 +86,7 @@ class BNote extends AbstractBeccaEntity<BNote> {
     /** number of note revisions for this note */
     private revisionCount!: number | null;
 
-    constructor(row: Partial<NoteRow>) {
+    constructor(row?: Partial<NoteRow>) {
```

Comment:
In `becca_loader.ts` we have:

```js
new BNote().update(row).init();
```

Please note that the null check does not throw, it is just a guard condition to prevent `updateFromRow` and `init` if called with an empty constructor.